tests/test_all.py

```diff
@@ -1,18 +1,30 @@
-import random
+"""
```

Comment:
Three tests fail for me locally:

```
FAILED tests/test_all.py::test_set_history_doc_refs_empty_final_url_valid_uuid
FAILED tests/test_all.py::test_set_history_doc_refs_existing_by_url_with_wrong_dealroom_uuid
FAILED tests/test_all.py::test_set_history_doc_refs_for_deleted_company_w_uuid_2
```

Reply:
The only failing test for me was `test_set_history_doc_refs_existing_by_url_with_wrong_dealroom_uuid`.

I fixed it in https://github.com/dealroom/data-firestore-connector/pull/31/commits/570f346423c82813d411c681f88ad9b15ffee66d (please read the description in the commit to understand why that change was made)